lib/user.js

```diff
@@ -117,9 +121,22 @@ function access(permission, account, callback) {
       callback(err);
       return;
     }
-
-    callback(null, permissions.indexOf(permission) > -1);
+    findPermission(permissions);
```

Comment:
Same as above:

This could return a boolean captured in a local ´permissionFound´ variable and passed to the ´callback()´